src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,313 @@
+        firstParam = ZERO_VALUE;
+        secondParam = SECOND_POSITIVE_VALUE;
+        double expected = firstParam * secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '*');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void multiplicationZeroAtB() {
+        firstParam = FIRST_POSITIVE_VALUE;
+        secondParam = ZERO_VALUE;
+        double expected = firstParam * secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '*');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void multiplicationMinAndMaxValues() {
+        firstParam = Integer.MAX_VALUE;
+        secondParam = Integer.MIN_VALUE;
+        double expected = firstParam * secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '*');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void divisionTwoPositive() {
+        firstParam = FIRST_POSITIVE_VALUE;
+        secondParam = SECOND_POSITIVE_VALUE;
+        double expected = firstParam / secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '/');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void divisionTwoNegative() {
+        firstParam = FIRST_NEGATIVE_VALUE;
+        secondParam = SECOND_NEGATIVE_VALUE;
+        double expected = firstParam / secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '/');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void divisionPositiveAndNegative() {
+        firstParam = FIRST_NEGATIVE_VALUE;
+        secondParam = SECOND_POSITIVE_VALUE;
+        double expected = firstParam / secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '/');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void divisionZeroAtA() {
+        firstParam = ZERO_VALUE;
+        secondParam = SECOND_POSITIVE_VALUE;
+        double expected = firstParam / secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '/');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void divisionZeroAtB() {
+        firstParam = FIRST_POSITIVE_VALUE;
+        secondParam = ZERO_VALUE;
+        assertThrows(RuntimeException.class,
+                () -> calculator.calculate(firstParam, secondParam, '/'));
+    }
+
+    @Test
+    public void divisionMinAndMaxValues() {
+        firstParam = Integer.MAX_VALUE;
+        secondParam = Integer.MIN_VALUE;
+        double expected = firstParam / secondParam;
+        double actual = calculator.calculate(firstParam, secondParam, '/');
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void raisingPositiveToPositive() {
+        firstParam = FIRST_POSITIVE_VALUE;
+        secondParam = SECOND_POSITIVE_VALUE;
+        double expected = (int) Math.pow(firstParam, secondParam);
```

Comment:
```suggestion
        double expected = Math.pow(firstParam, secondParam);
```
same for all division tests and in solution